website/docs/r/database.html.markdown

```diff
@@ -32,8 +32,19 @@ resource "ibm_database" "<your_database>" {
   tags              = ["tag1", "tag2"]
 
   adminpassword                = "password12"
-  members_memory_allocation_mb = 3072
-  members_disk_allocation_mb   = 61440
+
+  group {
+    group_id = "member"
+    
+    memory { 
+      allocation_mb = 14336
+    }
+    
+    disk { 
+      allocation_mb = 20480
+    }
```

Comment:
We should make sure these allocations are same as the "Small" T-Shirt sizes.
It will be different for each database type.

Reply:
Done :D